Sources/TuistKit/ProjectGenerator/ProjectGenerator.swift

```diff
@@ -109,7 +109,9 @@ class ProjectGenerator: ProjectGenerating {
         try lint(graph: graph)
 
         // Generate
-        let updatedWorkspace = workspace.merging(projects: Array(graph.projects.map { $0.path }))
+        var projectPaths = Array(graph.projects.map { $0.path })
+        projectPaths.append(contentsOf: Array(Set(graph.projects.flatMap { $0.podProjectPaths })))
```

Comment:
looks like we need a `WorkpsaceMapper` :)

Reply:
I believe this isn't necessary, as `pod install` will re-write the workspace / projects anyways - CocoaPods has to have final say on the workspace for the integration to work 🤐

Reply:
ok I think I get the motivation behind this - this will indeed help fix #1226

I updated the fixture `ios_app_with_pods` and moved the pods projects to `additionalFiles` (we have to treat them as opaque projects)